ground/gcs/src/plugins/config/inputchannelform.cpp

```diff
@@ -210,6 +210,9 @@ void inputChannelForm::groupUpdated()
         case ManualControlSettings::RSSITYPE_TBSCROSSFIRE:
             count = 12;
             break;
+          case ManualControlSettings::RSSITYPE_IBUS:
```

Comment:
Please remove 2 spaces here so it lines up with the others. Otherwise, LGTM.

Reply:
Sorry I missed this